delete-user-data/extension.yaml

```diff
@@ -15,7 +15,7 @@
 name: delete-user-data
 displayName: Delete User Data
 specVersion: v1beta
-version: 0.2.0
+version: 0.1.2
```

Comment:
Bumped down, because originally we thought this would be a breaking change, but we made it non-breaking by making the default of the param "shallow". So now it's just 0.1.1 > 0.1.2